documentation/book/getting-started.adoc

```diff
@@ -9,12 +9,10 @@ The Docker images are available on {DockerRepository}.
 
 === Prerequisites
 
-An {ProductPlatformName} cluster is required to deploy {ProductLongName}. {ProductName} supports all kinds of clusters - from public and
```

Comment:
As I understand we have two different ProductPlatformName: OpenShift and Kubernetes. One word begins with a vowel "O" and the other a consonet "K" which should follow an article "an" and an article "a" respectively ( e.g. An Openshift cluster, a Kubernetes cluster ). Since the article only depends on the word immediately following itself, I suggest we write replace s/an [ProductPlatformName]/a running [ProductPlatformName]/g or some other adjective in front of ProductPlatormName in order to keep the article from needing to change